tool/release/Bump.kt

```diff
@@ -0,0 +1,66 @@
+package tool.release
+
+import com.google.api.client.http.ByteArrayContent
+import com.google.api.client.http.GenericUrl
+import com.google.api.client.http.HttpHeaders
+import com.google.api.client.http.javanet.NetHttpTransport
+import java.nio.file.Files
+import java.nio.file.Paths
+
+
+fun postJson(url: String?, authorization: String, accept: String, content: String) {
+    NetHttpTransport()
+    .createRequestFactory()
+    .buildPostRequest(GenericUrl(url), ByteArrayContent.fromString("application/json", content))
+    .setHeaders(HttpHeaders().setAuthorization(authorization).setAccept(accept))
+    .execute()
+}
+
+fun increaseVersion(version: String): String {
```

Comment:
can we call it `bumpVersion` (I think increase and bump has the same meaning in this context). We want to always reduce the number of terminology to the minimum required.

Reply:
I think the algorithm can be simplified:

```
fun bumpVersion(version: String): String {
  current = version.split('.').getLast()
  current = current.split('-').getLast()
  try {
    next = parseInt(current) + 1
  }
  catch (e: NumberFormatException) {
    throw RuntimeException('unparseable ...')
  }
}

Reply:
where's the case where you handle version like `2.0.0`?

Reply:
@lolski addressed: https://github.com/graknlabs/dependencies/pull/237/commits/1a9419eb196abf2a85a6df22fa7261dbe7eccf78

Reply:
@lolski it's handled in the very first line (as it's the most basic case):

```
        (Integer.parseInt(version) + 1).toString()
```